resources/geocoding/en/52.txt

```diff
@@ -0,0 +1,819 @@
+5255|Mexico City
```

Comment:
Thanks!

a) We need the file to have a copyright header, like all the other geocoding files in this directory
b) What about a Spanish variant, since that is the local language? Although we can add this ourselves if you don't have familiarity with Mexico/are busy.
c) +5255 should be in the middle, the file isn't ordered by prefix length, but numerically, so it should go after line 311.
d) Where does this data come from? This needs to be in the header too.